frontend/ui/widget/imageviewer.lua

```diff
@@ -398,16 +402,15 @@ function ImageViewer:_new_image_wg()
 
     local rotation_angle = 0
     if self.rotated then
-        -- in portrait mode, rotate according to this global setting so we are
-        -- like in landscape mode
-        -- NOTE: This is the sole user of this legacy global left!
-        local rotate_clockwise = G_defaults:readSetting("DLANDSCAPE_CLOCKWISE_ROTATION")
+        -- The default is to rotate clockwise when in portrait mode, so right handed users
+        -- get to hold the device's bottom in their right hand.
```

Comment:
Hmm, if you rotate clockwise, it's what used to be the device's *top* edge (when in Portrait) that ends up in your right hand?

Reply:
(Assuming we're talking about *device* rotations, and not buffer rotations, which seems to hold true given the code below ;))

Reply:
In that case, yeaaaah, I'd probably ask to *invert* the text (i.e., CW -> CCW), because everything else in the UI assumes we're talking about device rotation (similarly, the icons no longer make any sense now tyhat you've clrafieid that).

Reply:
I don't care about the icons. (I understand the black thingie in mine could be thought of as the device or as the image, it's as confusing as I wrote in my first post :).
If you think the rounded arrows, being more abstract, makes it less confusing, fine with me. (Which pair then, the 180° or 360° ones?)

But even if it's in the "Rotation>" mixed of all rotation stuff submenu, it's still about rotation when in ImageViewer, which itself does not rotate: only the image in it is rotated.
So, CW / CCW meaning how one should rotate the device to see the original image - while having the ImageViewer's buttons' text vertical on the side and unreadable - feels... a lot more odd to me.

Reply:
If we can't agree on the meaning of Clockwise/Counterclockwise (given that they would be that only in portrait mode, and the reverse when in landscape), what about just the most natural (I think, nobody mention they agreed with me :)) for each population:
```
[x] Right handed
[ ] Left handed
```

Reply:
> Then an explicit clarification that it's an _image_ rotation?

Which may or may not be implied by the context, so, eeeeh ;D.

Reply:
> > Then an explicit clarification that it's an _image_ rotation?
> 
> Which may or may not be implied by the context, so, eeeeh ;D.

The help text is perfectly clear, so maybe just talking about the upper-level menu entry there... but I can't think of anything that doesn't sound like ass, so, eeeh, might be fine; the icons are what got me ;).

Reply:
It's currently:
![image](https://github.com/koreader/koreader/assets/24273478/85cd1b35-952a-40df-9ecd-85917676f868)
You mean it would be clearer if it was `Image viewer's image rotation >` ?
Or the submenu item:
```
[x] Rotate image clockwise
[ ] Rotate image counterclockwise
```
?

Reply:
I can't think of anything that doesn't sound awful for the top-level menu, so if the actual "rotate" entries sound fine that way, yeah, that'll do ;).

Reply:
> (I think, nobody mention they agreed with me :))

I would tend to rotate the other way around, so, I'd stick with angles ;p.

Reply:
Forgot the "in portrait mode" :) and the fact that it's inverted when in landscape :/

`[x] Rotate image clockwise when device in portrait, counter clockwise when in landscape`
which is too long...

That behaviour was already there - I don't remember if it was my decision at the time, but I kinda understand and like it: if the user is reading with device in landscape, and there is a tall/portrait image, rotating it to see it bigger, it feels natural that the use should rotate its device so the device bottom is at the bottom.
But it contradicts my tilt/pendulum/gravity UX I described above :)

Reply:
I'm fine with just tweaking the icons ;).

Reply:
I you're having a string length concern, it's not particularly pretty but the 180° arrows can kinda replace CW & CCW *inline*?

Reply:
e.g., `Rotate image U+whatever blah blah`

Reply:
(Is "when in portrait" Englishly correct enough?)

Reply:
I guess that at this point, we could have 4 menu items :)
```
[x] Rotate image CW when in portrait
[ ] Rotate image CCW when in portrait
----
[ ] Rotate image CW when in landscape
[x] Rotate image CCW when in landscape
```

Reply:
> (Is "when in portrait" Englishly correct enough?)

Possibly something along the lines of "in portrait mode" might be better understood?

Reply:
> I guess that at this point, we could have 4 menu items :)

That's is indeed also an option (and perhaps easier on I18N ;)).